vertical-pod-autoscaler/pkg/recommender/input/history/history_provider.go

```diff
@@ -17,23 +17,29 @@ limitations under the License.
 package history
 
 import (
+	"context"
 	"fmt"
-	"net/http"
 	"sort"
 	"strings"
 	"time"
 
+	promapi "github.com/prometheus/client_golang/api"
```

Comment:
nit: I think it's cleaner to have the github.com imports in one group and the k8s.io/autoscaler ones in the other.